packages/framework/collabspaces/src/contracts.ts

```diff
@@ -84,4 +84,16 @@ export interface IEfficientMatrixTest {
 
 	// Returns a structure with various debug info about the cell
 	getCellDebugInfo(row: number, col: number): Promise<{ channel?: ICollabChannelCore }>;
+
+	getCellChannelIdDebugInfo(row: number, col: number): Promise<{ channelId: string }>;
+
+	getReverseMapsDebugInfo(
+		rowId: string,
+		colId: string,
+	): {
```

Comment:
I'm not sure returning rowMapSize & colMapSize is useful - I'd rather add asserts across all of the code (including this API call) that they are the same as size of the matrix,
I'd also probably not use word "Index" as we do not use it on other APIs - a bit easier to connect the dots.

Reply:
My goal was to automatically tell whether the reverse mapping matrixes were updated using their size as a reference.